rai_test_utils/requirements.txt

```diff
@@ -5,3 +5,5 @@ scipy
 shap
 lightgbm
 xgboost
+torch==1.13.1
+torchvision==0.14.1
```

Comment:
Could you maybe add these dependencies as extras for his package? May be add extra like object_detection. `pip install rai_test_utils[object_detection]`.

More on extras here:- https://stackoverflow.com/questions/52474931/what-is-extra-in-pypi-dependency